fixture-monkey-jakarta-validation/src/main/java/com/navercorp/fixturemonkey/jakarta/validation/introspector/JakartaValidationJavaArbitraryResolver.java

```diff
@@ -117,17 +116,31 @@ public Arbitrary<String> strings(
 		return arbitrary
 			.filter(it -> {
 				if (!notBlank) {
-					return true;
+					if (it == null) {
+						return true;
+					} else {
+						return !containsControlCharacters(it);
+					}
 				}
 
-				if (it == null || it.trim().length() == 0) {
+				if (it == null || it.trim().isEmpty()) {
 					return false;
 				}
 
-				return !CONTROL_BLOCK.equals(it.trim());
+				return !containsControlCharacters(it);
 			});
 	}
 
+	private static boolean containsControlCharacters(String value) {
+		for (char c : value.toCharArray()) {
+			if (Character.isISOControl(c)) {
+				return true;
+			}
+		}
+
+		return false;
+	}
+
```

Comment:
This is a private method that uses a built-in function of the `Character` class to return whether this character is a control character.